### PR TITLE
Upgrade redis

### PR DESCRIPTION
**Pull request by AnnaKL**
Upgrade redis version to the latest.

### CODE REVIEW

File: reliable_queue.gemspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.files = Dir['*.{md,txt}', 'lib/**/*.rb']
   s.test_files = Dir['spec/**/*.rb']
 
-  s.add_dependency 'redis', '~> 3.3'
+  s.add_dependency 'redis', '~> 4.4'
```
Comment (jbilbo) [inline reliable_queue.gemspec]: shall we allow also people who are still (for whatever reason) in the 3.x?

```ruby
s.add_dependency 'redis', '>= 3.0.0'
```

**Review by jbilbo — APPROVED** [review]

File: reliable_queue.gemspec (hunk 1 above)
Comment (AnnaKL) [reply reliable_queue.gemspec]: I was thinking about it. Is it not risky though? If they release redis version not compatible at some point we might be in trouble right? Or am I too cautious ?

Comment (jbilbo) [reply reliable_queue.gemspec]: Hmm not sure what's the best practice here. I guess we could walk the cautious path and:

```ruby
s.add_dependency 'redis', '>= 3.0.0', '< 5.0.0'
```

Comment (AnnaKL) [reply reliable_queue.gemspec]: Me neither but this feels like the best solution. I will apply it.

**Pull request merged**